views/pwas/list.hbs

```diff
@@ -24,11 +24,15 @@
             {{#each pwas}}
             <div class="media">
                 <a href="/pwas/{{id}}">
-                    <div class="media-left"><img src="/pwa.png">
+                    <div class="media-left">
+                        <svg width="64" height="64">
+                            <rect width="64" height="64" stroke-width="4" fill="{{backgroundColor}}" />
+                            <text x="32" y="32" fill="white" alignment-baseline="middle" text-anchor="middle" font-size="32px">{{firstletter}}</text>                            
```

Comment:
Not completely sure about this, but it might be better to use a [helper](http://handlebarsjs.com/#helpers) for the `firstletter` stuff, so that this would turn into something like `{{firstLetter name}}`. Maybe depends on what other transformations we want to do.